### PR TITLE
Add platform details to release notes

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

Added platform details to release notes.

### CODE REVIEW

File: docs/platform/release-notes/tizen-5-5-m1.md [hunk 1]
```diff
@@ -1,11 +1,26 @@
-# Tizen 5.5 Public M1 Release Notes
+# Tizen 5.5 Public M1
 
-Release Date: May. 31, 2019
+Release date: May 31, 2019
```
Comment (mijin85cho) [inline docs/platform/release-notes/tizen-5-5-m1.md]: Question

"Release date:"
 - Tizen 5.5 Public M2
 - Tizen 5.5 Public M1
 - Tizen 5.0 Public M2
 - Tizen 5.0 Public M1

"Release Date : "
 - Tizen 4.0 Public M3
 - Tizen 4.0 Public M2
 - Tizen 4.0 Public M1

Are we going to use small letter for "date" from 5.0  versions?

File: docs/platform/release-notes/tizen-5-5-m1.md [hunk 2]
```diff
@@ -592,31 +607,31 @@ Release Date: May. 31, 2019
   - New InputMethod API set to update the input panel event has been added.
   - New Application API set to add CurrentDeviceOrientation property for CoreUIApplication has been added.
 
-### Known Issues
+#### Known Issues
 
 - Xamarin.Forms
   - For more information on the list of limitations, see [here](https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations).
```
Comment (mijin85cho) [inline docs/platform/release-notes/tizen-5-5-m1.md]: Don't we need to use the link of docs.tizen.org, not developer.tizen.org?

File: docs/platform/release-notes/tizen-5-0-m2.md [hunk 3]
```diff
@@ -1,476 +1,491 @@
...
+  - Added new InputMethod API set for floating keyboard and to hide or launch IME.
+  - Added new Wi-Fi API set for forgetting access point (AP), BssidScan, RawSsid, CountryCode, and WPS connection cancellation.
+  - Added new Applications API set for rpc-port to support Remote Procedure Call (RPC).
+  - Added new NUI API set for KeyboardRepeatInfo and TextPredition.
+  - Added the change of NUI API set for ChildAdded, ChildRemoved, PropertySet events and properties of TableView.CellPosition.
+
+#### Known Issues
+
+- Xamarin.Forms
+  - For more information on the list of limitations, see [here](https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations).
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/platform/release-notes/tizen-5-0-m2.md]: URL was changed.

File: docs/platform/release-notes/tizen-4-0-m2.md [hunk 4]
```diff
@@ -1,9 +1,10 @@
 # Tizen 4.0 Public M2
 
-Release Date: 01 Nov, 2017
+Release date: Nov. 11, 2017
```
Comment (mijin85cho) [inline docs/platform/release-notes/tizen-4-0-m2.md]: Question. Nov. 01 is changed to Nov. 11.
But it looks like Nov. 01 is correct. 
Would you please check?

File: docs/platform/release-notes/tizen-4-0-m1.md [hunk 5]
```diff
@@ -1,10 +1,11 @@
 # Tizen 4.0 Public M1
 
-Release Date: 31 May, 2017
+Release date: May 31, 2017
```
Comment (mijin85cho) [inline docs/platform/release-notes/tizen-4-0-m1.md]: May. (Add a period at the end.)

File: docs/platform/release-notes/tizen-2-1.md [hunk 6]
```diff
@@ -1,8 +1,6 @@
 # Tizen 2.1
 
-Release Date: 18 May, 2013
-
-
+Release date: May 18, 2013
```
Comment (mijin85cho) [inline docs/platform/release-notes/tizen-2-1.md]: May. (Add a period at the end.)

File: docs/platform/release-notes/tizen-5-0-m2.md [hunk 7]
```diff
@@ -1,476 +1,491 @@
...
+
+#### New and changed features
+
+- New API set and features
+  - D-Bus
+    - A policy rule checker has been developed and deployed for improving security.
+  - Dlog
+    - A new feature of front-end filter has been developed for efficient logging and facilitating runtime log analysis.
+  - Deviced
+    - C## API set for display control has been added.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/platform/release-notes/tizen-5-0-m2.md]: All C## were fixed.

File: docs/platform/release-notes/tizen-4-0-m2.md (hunk 4 above)
Comment (journey2w) [reply docs/platform/release-notes/tizen-4-0-m2.md]: Fixed.

File: docs/platform/release-notes/tizen-4-0-m1.md (hunk 5 above)
Comment (journey2w) [reply docs/platform/release-notes/tizen-4-0-m1.md]: Nov. 01, 2017 is November 01, 2017, but May is May. In other words, May 31, 2017.

**Pull request merged**